src/lib/components/collections/collection-tab/MyCollection.svelte

```diff
@@ -11,7 +11,7 @@
   import type { CollectionListViewModel } from "../collections-list/CollectionList.ViewModel";
   import type { CollectionDocument } from "$lib/database/app.database";
   import type { Observable } from "rxjs";
-  import type { WorkspaceRole } from "$lib/utils/enums";
+  import { ResponseStatusCode, type WorkspaceRole } from "$lib/utils/enums";

```

Comment:
why is one enum imported as `type` and the other is imported without `type` keyword?

Reply:
type not required, removed